### ISSUE TITLE
How to acquire inertia from sklearn-som when using elbow method

### ISSUE THREAD

**Issue by hendrywijaya98** [rileypsmith/sklearn-som#4]
as you know as usual when you using kmeas from scikit-learn, there's some method provided like in this case that we calling `inertia_` attribute from Kmeans

`inertia_values = []
mapping = {}
K = range(1,15)

for k in K:
    som = SOM(m=k, n=1, dim=7)
    som = som.fit(X)
    inertia_values.append(som.inertia_)
    mapping[k] = som.inertia_

for key, val in mapping.items():
    print(f'{key} : {val}')`

so i'm try the same way in sklearn-som, but the result is on screenshot below

![image](https://user-images.githubusercontent.com/57217977/116892140-b4c7ec80-ac59-11eb-90aa-c71668e86c16.png)

so, in sklearn-som how we can call inertia just like from kmeans does

best regards

**Comment by rileypsmith** [comment]
Hi Henry, please note that the .fit() method is an in-place method and does not actually return anything, just like most classifiers in sklearn. So when you do `som = som.fit(X)` it will actually set `som = None` since the .fit() method does not return anything. That is why you get the error `'NoneType' object has no attribute 'inertia_'` when calling `som.inertia_`.

Please try this instead:
```py
som = SOM(m=k, n=1, dim=7)
som.fit(X)
inertia_values.append(som.inertia_)
mapping[k] = som.inertia_
```
I think that should work for you.
